external-docs/docs/guides/shape-data.md

```diff
@@ -1,3 +1,241 @@
 # Shaping your data
 
-todo: Explain the shapes of data we accept. Issue #17.
+> This page discusses the shapes of input data supported by csvcubed with the aim to help you understand which to use and how to generate data in the right shape.  
+
+csvcubed requires that all CSV data inputs are provided in one of two specialised form of [tidy data](../glossary/index.md#tidy-data):
+
+* the [canonical approach](#canonical-shape) - the **default recommended shape** accepted by csvcubed. It is the **most flexible** but also the most **verbose**.
```

Comment:
Why is it the most flexible? Can we outline a reason or put a reference here? Could be helpful for the unfamiliar users.